apps/namada-interface/src/App/TopNavigation/topNavigation.tsx

```diff
@@ -315,7 +315,10 @@ function TopNavigation(props: TopNavigationProps): JSX.Element {
               </LogoContainer>
               <ColorModeContainer>
                 {topLevelRoute !== TopLevelRoute.StakingAndGovernance && (
-                  <Toggle checked={isLightMode} onClick={toggleColorMode} />
+                  <Toggle
+                    checked={colorMode == "dark"}
```

Comment:
I think it would always best to default to strict equality when there is no special need for the abstract equality. I saw this slipped in already at the previous PR, so maybe would be still good to clean that up.
```suggestion
                    checked={colorMode == "dark"}
```